app/include/bfx-api-cpp/BitfinexAPI.hpp

```diff
@@ -35,21 +35,15 @@
 #include <utility>
 #include <vector>
 
-// curl
-#include <curl/curl.h>
-
-// cryptopp
-#include <cryptopp/base64.h>
-#include <cryptopp/hex.h>
-#include <cryptopp/hmac.h>
-#include <cryptopp/osrng.h>
-
 // internal jsonutils
 #include "jsonutils.hpp"
 
 // internal error
 #include "error.hpp"
 
+// internal TRequest
+#include "TRequest.hpp"
```

Comment:
I though I'm clever but now I see I'm not. I can't figure out what means `T` in `TRequest.hpp` :). I would rename it to "two word name"  `T....Request.hpp`.

Reply:
Sorry for it. It is just my way to define `class`, so then we can instance with the name of the object. So in this case it is `TRequest` so then we can do:
```c++
TRequest Request = new TRequest();
Request.get(...);
```
So in my mind `T` is `Template` like a way to call the recipe of abstraction (class) of the object (instance). But of course we can name it as you prefer.

Reply:
I was looking in `TRequests.hpp` and I would rename all occurrences of `TRequest` to `HTTPRequest` which will improve code clarity.

Reply:
@MMquant sorry for the delay. I will change as soon as I have a moment.

Reply:
I'll change to `HTTPRequest` even if the request is not limited to the HTTP protocol.